changelog/next/features/5169--multi.md

```diff
@@ -0,0 +1,2 @@
+The `pipeline::detach` operator offers a convenient way of running a pipeline in
+the background at the node.
```

Comment:
Probably also not needed in the changelog, unless we want people to start using it?